modules/shared/components/molecules/CreatePostHeader/CreatePostHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { ICreatePostHeader } from './ICreatePostHeader';
+import Avatar from '../../atoms/Avatar/Avatar';
+import TabGroup from '../TabGroup/TabGroup';
+
+const CreatePostHeader: FC<ICreatePostHeader.IProps> = (
```

Comment:
we did not add a prefix in comment names`Create`      <CreateButton /> ?
we can commit the `Create` prefix
we need to follow same we name our components

Reply:
we have `PostViewHeader` so i thought it would be more explicit to name it `CreatePostHeader`
because `create` prefix refers to the post creation
what do you think ?

Reply:
um, ok `CreatePostHeader` is fine and `PostHeader`  instead of `PostViewHeader `